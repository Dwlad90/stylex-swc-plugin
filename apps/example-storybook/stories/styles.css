/* StyleX styles will be injected here by PostCSS */
@stylex;
