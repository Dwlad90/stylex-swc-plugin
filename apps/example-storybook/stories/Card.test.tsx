import { describe, it, expect } from 'vitest';
import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import * as CardStories from './Card.stories';


describe('Card Snapshots', () => {
  const stories = composeStories(CardStories);

  Object.entries(stories).forEach(([name, Story]) => {
    it(`renders ${name} story correctly`, () => {
      const { container } = render(<Story />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});

