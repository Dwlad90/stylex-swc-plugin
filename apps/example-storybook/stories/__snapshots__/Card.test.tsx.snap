// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card Snapshots > renders Default story correctly 1`] = `
<div
  class="backgroundColor-x12peec7 borderRadius-xur7f20 padding-x1tamke2 boxShadow-x1mh2kpm borderWidth-xmkeg23 borderStyle-x1y0btm7 borderColor-x159w2ql maxWidth-xw5ewwj"
  data-style-src="stories/Card.tsx:5"
>
  <h3
    class="fontSize-xosj86m fontWeight-x117nqv4 marginBottom-x1e56ztr color-x1525slw"
    data-style-src="stories/Card.tsx:15"
  >
    Card Title
  </h3>
  <p
    class="fontSize-xif65rj lineHeight-x1evy7pa color-x33gn0b"
    data-style-src="stories/Card.tsx:21"
  >
    This is some example content for the card component.
  </p>
</div>
`;

exports[`Card Snapshots > renders Elevated story correctly 1`] = `
<div
  class="backgroundColor-x12peec7 borderRadius-xur7f20 padding-x1tamke2 borderWidth-xmkeg23 borderStyle-x1y0btm7 borderColor-x159w2ql maxWidth-xw5ewwj boxShadow-xz3rqps"
  data-style-src="stories/Card.tsx:5; stories/Card.tsx:26"
>
  <h3
    class="fontSize-xosj86m fontWeight-x117nqv4 marginBottom-x1e56ztr color-x1525slw"
    data-style-src="stories/Card.tsx:15"
  >
    Elevated Card
  </h3>
  <p
    class="fontSize-xif65rj lineHeight-x1evy7pa color-x33gn0b"
    data-style-src="stories/Card.tsx:21"
  >
    This card has an elevated shadow effect.
  </p>
</div>
`;
